app/code/Magento/Store/App/Action/Plugin/StoreCheck.php

```diff
@@ -1,40 +1,46 @@
 <?php
 /**
- *
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
 namespace Magento\Store\App\Action\Plugin;
 
+use Magento\Framework\App\ActionInterface;
+use Magento\Framework\Exception\State\InitException;
+use Magento\Store\Model\StoreManagerInterface;
+
+/**
+ * Plugin verify Store on before Execute on ActionInterface
+ */
 class StoreCheck
 {
     /**
-     * @var \Magento\Store\Model\StoreManagerInterface
+     * @var StoreManagerInterface
      */
     protected $_storeManager;
 
     /**
-     * @param \Magento\Store\Model\StoreManagerInterface $storeManager
+     * @param StoreManagerInterface $storeManager
      */
     public function __construct(
-        \Magento\Store\Model\StoreManagerInterface $storeManager
+        StoreManagerInterface $storeManager
     ) {
         $this->_storeManager = $storeManager;
     }
 
     /**
-     * @param \Magento\Framework\App\Action\AbstractAction $subject
-     * @param \Magento\Framework\App\RequestInterface $request
+     * Verify before execute
+     *
+     * @param ActionInterface $subject
      * @return void
      * @SuppressWarnings(PHPMD.UnusedFormalParameter)
-     * @throws \Magento\Framework\Exception\State\InitException
+     * @throws \Magento\Framework\Exception\State\InitExceptionn
```

Comment:
I think it is a Typo

Reply:
It actually is.
Will fix this tomorrow.

Reply:
Fixed by myself